workflow/workflow.go

```diff
@@ -262,6 +262,24 @@ func SideEffect(ctx Context, f func(ctx Context) interface{}) encoded.Value {
 	return internal.SideEffect(ctx, f)
 }
 
+// MutableSideEffect executes provided function once, then it look up history for value with the given id,
```

Comment:
executes provided function -> executes the provided function
look up history -> looks up the history
for value -> for the value
given id, if there -> given id. If there